pkg/syncer/storagepool/service.go

```diff
@@ -132,26 +132,15 @@ func InitStoragePoolService(ctx context.Context,
 	go func() {
 		diskDecommEnablementTicker := time.NewTicker(common.DefaultFeatureEnablementCheckInterval)
 		defer diskDecommEnablementTicker.Stop()
-		clusterFlavor := cnstypes.CnsClusterFlavorWorkload
 		for ; true; <-diskDecommEnablementTicker.C {
```

Comment:
We don't need this ticker anymore?

Reply:
We do since ticker is used as a retry mechanism to if initializing Disk Decom Controller Fails.